staff-finding/experiments/gp_centerlines/gp_fitter.py

```
@@ -0,0 +1,160 @@
+        noise_level_bounds=noise_level_bounds,
+    )
+
+    gpr = GaussianProcessRegressor(
+        kernel=kernel,
+        n_restarts_optimizer=n_restarts,
+        normalize_y=True,
+    )
+
+    try:
+        gpr.fit(obs_x, obs_y)
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)